experimental/testdata/pthread.c

```diff
@@ -0,0 +1,18 @@
+#include <pthread.h>
```

Comment:
one last request.. if this isn't generated with `make build.examples.zig-cc`, can you add a source comment on when it can (e.g. link to zig or otherwise) and how to compile this meanwhile?

Reply:
Sure - I had the command documented in features_example_test.go following the pattern in listener_example_test.go, so I went ahead and added the link to zig there. If it's better to have in the source file itself though, I can either move it all here, or copy-paste it all here, let me know if that's better

Reply:
nope I missed that. all good.